docs/admin/installation/hardware.rst

```diff
@@ -534,56 +534,6 @@ startup and adjust the system configuration:
 
 .. |NUC10 leads| image:: ../../images/hardware/nuc10_leads.jpg
 
-.. _nuc8_recommendation:
```

Comment:
There should be language here to flag that these are EOL. Straight-up removing them from the docs is liable to cause confusion for folks still using them and looking for guidance.

For example "We previously recommended the <model> name. As it has reached EOL, we recommend replacing it with a model which still has vendor support. Compatibility testing for the grsec kernel will continue for the next N server releases" (where N is a number we argue about)

Reply:
I've changed it to be along these lines, but instead of promising an explicit sunset timeline, I've said it'll likely work but we're going to stop explicitly testing on it.